### PR TITLE
Added blog post ISMIR20, Ruchit

### PR DESCRIPTION
**Pull request by rragrawal**
Blog post about a paper from ISMIR 2020.

### CODE REVIEW

File: _posts/2020-12-02-sf_ismir20.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: “Learning to read and Follow music in Complete score sheet images”
+subtitle: Summary of an ISMIR 2020 paper on score following
+date: 2020-12-02
+author: Ruchit Agrawal
+author-id: ruchit
+background: /posts/sf_ismir20/sf.jpeg
+--- 
+This year's ISMIR was held virtually, following the trend of online conferences smitten by the COVID-bug.  I attended [ISMIR 2020](https://www.ismir2020.net/), the 21<sup>st</sup> International Society for Music Information Retrieval Conference, from the 11<sup>th</sup> to the 16<sup>th</sup> of October, 2020. I was delighted to find fascinating papers related to my current research. This blog post summarizes one such paper that piqued my interest. The paper is titled [“Learning to read and Follow music in Complete score sheet images”](https://arxiv.org/abs/2007.10736); and the authors are Florian Henkel, Rainer Kelz, and Gerhard Widmer, from JKU Linz, Austria.
```
Comment (furkanyesiler) [inline _posts/2020-12-02-sf_ismir20.md]: Fix the mixed "capital - non-capital" situation for the paper title.

Comment (furkanyesiler) [reply _posts/2020-12-02-sf_ismir20.md]: I.e., "Learning to read and Follow music in Complete score sheet images"

File: _posts/2020-12-02-sf_ismir20.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+The authors attempt to overcome this limitation by working with entire pages of sheet music images. They extend the work initially proposed in [^1] to be able to handle the temporal aspect of score following, i.e. disambiguate multiple positions in the sheet image that match with the audio excerpt. 
+
+## A brief detour: Referring image segmentation
+The basic idea is motivated by “referring image segmentation” (cite), a task in computer vision that aims to identify a region in an image, given a language expression as the query. To better understand this task, take a look at this figure.
```
Comment (furkanyesiler) [inline _posts/2020-12-02-sf_ismir20.md]: I assume you took a personal note with "(cite)". Please complete that citation.

File: _posts/2020-12-02-sf_ismir20.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+<figure class="figure w-100">
+  <img src="{{ '/posts/sf_ismir20/referring.png' | relative_url }}" alt="Referring image segmentation" class="figure-img img-fluid mx-auto d-flex">
+  <figcaption class="figure-caption text-center">
+  Referring image segmentation. Figure from [^3]
```
Comment (furkanyesiler) [inline _posts/2020-12-02-sf_ismir20.md]: The citation [^3] doesn't work in the caption. I'm not sure if it's possible to fix it.

File: _posts/2020-12-02-sf_ismir20.md (hunk 2 above)
Comment (furkanyesiler) [reply _posts/2020-12-02-sf_ismir20.md]: And maybe finish the last sentence with ":".

File: _posts/2020-12-02-sf_ismir20.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  The U-net architecture employed in the paper
+  </figcaption>
+</figure>
+
+
+A feature-wise linear modulation layer [^4] is also applied before the last activation function, to directly interfere with the learned representation of the sheet image by modulating its feature maps. This helps the CNN to focus on the parts required for segmentation. The FiLM layer is illustrated below:
+<figure class="figure w-100">
+  <img src="{{ '/posts/sf_ismir20/film.png' | relative_url }}" alt="FiLM layer" class="figure-img img-fluid mx-auto d-flex">
+  <figcaption class="figure-caption text-center">
+  The FiLM layer used before the last activation [^4]
```
Comment (furkanyesiler) [inline _posts/2020-12-02-sf_ismir20.md]: The citation [^4] doesn't work here either.

**Review by furkanyesiler — CHANGES_REQUESTED** [review]
Some minor work needed.

**Comment by rragrawal** [comment]
Fixed the typos and other minor issues.

**Pull request merged**